doc/api/addons.markdown

```diff
@@ -44,7 +44,7 @@ be used as a starting-point for your own Addon.
 This "Hello world" example is a simple Addon, written in C++, that is the
 equivalent of the following JavaScript code:
 
-    module.exports.hello = function() { return 'world'; };
+    module.exports.hello = () => 'world';
```

Comment:
@silverwind if you'd like I can just fix that in the PR, I didn't want to make any opinionated changes though.